hack/update-crd-validation/main.go

```diff
@@ -0,0 +1,143 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"reflect"
+	"strings"
+
+	"k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+	crdgenerator "sigs.k8s.io/controller-tools/pkg/crd/generator"
+
+	_ "github.com/openshift/api"
+)
+
+var (
+	scheme = runtime.NewScheme()
+	codecs = serializer.NewCodecFactory(scheme)
+)
+
+func init() {
+	v1beta1.AddToScheme(scheme)
+}
+
+func main() {
+	// load existing manifests from manifests/ dir
+	existingManifests, err := crdsFromDirectory("manifests")
```

Comment:
I'm ok starting here.   Seems like we should just generate everything though

Reply:
what else?

Reply:
We have to copy this mechanism to all repos that have CRDs.